.gitignore

```diff
@@ -25,3 +25,7 @@ tests/cases/*.rs
 tests/cases/*.cpp
 tests/cases/*.kt
 tests/cases/*.nim
+dist/
+pyjl/test_files
+test.sh
+tests/output
```

Comment:
`pyjl/test_files` and `test.sh` dont belong in here (yet?).  

`tests/output` is useful, but could you add an end-of-line (EOL) at end-of-file (EOF) here. github and other tools show missing EOL as an error